azurerm/internal/services/policy/policy_assignment_resource.go

```diff
@@ -123,10 +126,40 @@ func resourceArmPolicyAssignment() *schema.Resource {
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+
+			"metadata": {
```

Comment:
could we [add this new field to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/policy_assignment.html.markdown#L89-L93)?

Reply:
documentation updated